src/bo4e/enum/marktrolle.py

```diff
@@ -1,23 +1,21 @@
+"""
+Diese Rollen kann ein Marktteilnehmer einnehmen.
+"""
 from enum import Enum
 
-
-class Marktrolle(str, Enum):
-    """
-    Diese Rollen kann ein Marktteilnehmer einnehmen.
-    """
-
-    VNB = "NB"  # Verteilnetzbetreiber
-    LF = "LF"  # Lieferant
-    MSB = "MSB"  # metering service operator
-    DIENSTLEISTER = "DL"  # unspecified service provider
-    BKV = "BKV"  # Bilanzkreisverantwortlicher,
-    BIKO = "BKV"  # Bilanzkoordinator / Marktgebietsverantwortlicher
-    UENB = "UENB"  # Übertragungsnetzbetreiber
-    KUNDE_NN_SELBST = (
-        "KUNDE-NN-SELBST"  # Kunden, die Netznutzungsentgelte selbst zahlen
-    )
-    MGV = "MGV"  # Marktgebietsverantwortlicher
-    EIV = "EIV"  # Einsatzverantwortlicher
-    RB = "RB"  # Registerbetreiber
-    KUNDE = "KUNDE"
-    INTERESSENT = "INTERESSENT"
+_marktrolle = {
+    "VNB": "NB",  # Verteilnetzbetreiber
```

Comment:
Wie kommt ihr eigenlich darauf, dass das ein Verteilnetzbetreiber ist? Im B04E steht nur NB - Netzbetreiber

Reply:
"common sense". meistens meint man, wenn man nicht explizit ÜNB sagt den Verteilnetzbertreiber, weil der traditionell sehr viel stärker, z.B. in die Marktkommunikation eingebunden ist als der ÜNB.